### ISSUE TITLE
Xmr-stak 2.5.2 release (NVIDIA Bug fix)

### ISSUE THREAD

**Issue by cartesien** [papampi/nvOC_miners#25]
@psychocrypt psychocrypt released this 2 days ago
Assets 3

xmr-stak-win64-2.5.2.zip
33.2 MB
Source code
(zip)

    Source code
    (tar.gz)

This release fixes a NVIDIA bug which reduce the hash rate for monero/cryptonight_v8 a little bit.
The upcoming Graft fork on 31st Oct to the POW cryptonight_v8 is supported with this release.

Config files from 2.5.X are compatible and 2.4.X or older are NOT compatible to this release.
Changelog:

    NVIDIA: fix wrong number of threads #2003
    Update for upcoming Graft CNv2 fork #2001

https://github.com/fireice-uk/xmr-stak/tree/2.5.2

**Comment by psychocrypt** [comment]
What is the issue? I do not get whats the problem?

**Comment by psychocrypt** [comment]
Ahh sry have not seen that this is a fork.

**Comment by cartesien** [comment]
@psychocrypt 

I did not necessarily want to warn you here. It's a copy-paste of your explanatory text from your github.
I'll be careful next time not to alert you unnecessarily

I posted this issue for lukepicci to update the software in nvOC.
Papampi made the necessary with lukepicci.

I worked this way because it is the method they asked me to practice in case of update of a mining software in nvOC.

Translated from French with Google translation and not re-read for correction.
